cnf-certification-test/identifiers/identifiers.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (C) 2021-2022 Red Hat, Inc.
+// Copyright (C) 2021-2023 Red Hat, Inc.
```

Comment:
So, what's the point of only updating the year in those three files? What about the rest?

Reply:
@rdavid Do you want to add it as a "linter" github workflow?

Reply:
@sebrandon1 It is a cool idea. I'll take a look, maybe the script needs to adopt.